boards/nucleo-f070/board.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2016 Inria
```

Comment:
Normally, now it's Inria, according to [this](https://www.inria.fr/en/content/view/full/11297) (look at the `Reminder` section on the right column).